app/templates/partials/ga.html

```diff
@@ -0,0 +1,10 @@
+<!-- Google tag (gtag.js) --> 
+<script async src="https://www.googletagmanager.com/gtag/js?id={{ gtm_tag_id }}"></script> 
+
+<script> 
```

Comment:
I think putting `nonce="{{ cspNonce }}` after `<script ` should fix the console error